src/main/io/vtx_smartaudio.c

```diff
@@ -570,11 +570,32 @@ static void saGetPitFreq(void)
 }
 #endif
 
+#define SA_LOWRACE_BAND 5
+
+uint8_t saGetBand(int8_t deviceChannel)
+{
+    return deviceChannel / 8;
+}
+
+uint8_t saGetChannel(int8_t deviceChannel)
+{
+    uint8_t band = deviceChannel / 8;
+    uint8_t channel = deviceChannel % 8;
+
+    // 'Low Race' band channels are in reversed order on TBS Unify for some reason
+    if (band == SA_LOWRACE_BAND) return 7 - channel;
+
+    return channel;
+}
+
 void saSetBandAndChannel(uint8_t band, uint8_t channel)
 {
     static uint8_t buf[6] = { 0xAA, 0x55, SACMD(SA_CMD_SET_CHAN), 1 };
 
-    buf[4] = band * 8 + channel;
+    // 'Low Race' band channels are in reversed order on TBS Unify for some reason
+    uint8_t fixedChannel = (band == SA_LOWRACE_BAND ? 7 - channel : channel);
+
+    buf[4] = band * 8 + fixedChannel;
```

Comment:
This should be probably wrapped in function - there are already `saGetChannel` and `saGetBand` functions, so implement inverse function in similar way (`saBuildDeviceChannel` ?)

Reply:
The intention was to maximally hide from user this fixup of low race band channels into most low-level channel/band functions. To do this, I had to decompose all the `saDevice.channel` "reads" into `saGetBand`/`saGetChannel`. On the other side, all the "writes" are happening at a single entry point —  `saSetBandAndChannel`, so I don't have to decompose anything. If I decompose this fixup into separate function, it would have a single usage and it would be possible to call this function — we definitely don't want this, since there would be no other "write" operations that require device channel value fixup.

Reply:
Personally I prefer orthogonality, with bidirectional mapping between `deviceChannel` and `band` + `channel`. Other consumers may find useful to 'pack' band+channel for storage (working with `deviceChannel` as opaque object with accessor functions)

But your argument is reasonable.